_data/navigation.yml

```
@@ -1,179 +1,28 @@
-- pages:
-    - title: About Swift
-      url: /about/
-      sections:
-        - title: Features
-          name: features
-        - title: Swift.org and Open Source
-          name: swiftorg-and-open-source
-        - title: Platform Support
-          name: platform-support
-    - title: Blog
-      url: /blog/
-    - title: Getting Started
-      url: /getting-started/
-      sections:
-        - title: Installing Swift
-          name: installing-swift
-        - title: Using Swift
-          name: using-swift
-        - title: Go Further
-          name: go-further
-    - title: Install
-      url: /install/
-      sections:
-        - title: macOS
-          name: macos
-        - title: Linux
-          name: linux
-        - title: Windows
-          name: windows
-    - title: Download
-      url: /download/
-      sections:
-        - title: Releases
-          name: releases
-        - title: Snapshots
-          name: snapshots
-        - title: Installing Swift
-          name: installing-swift
-    - title: Packages
-      url: /packages/
-      sections:
-        - title: Package Ecosystem
-          name: package-ecosystem
-        - title: Create Your Own
-          name: create-your-own
-        - title: Advanced Search
-          name: advanced-search
-    - title: Platform Support
-      url: /platform-support/
-      sections:
-        - title: Deployment and Development
-          name: deployment-and-development
-        - title: Deployment-only
-          name: deployment-only
-        - title: Platform Owners
-          name: platform-owners
+- header: Get Started
+  url: /getting-started/
+- header: Packages
+  url: /packages/
+  pages:
+    - title: Community Showcase
+      url: /packages/showcase.html
+    - title: Packages with Macros
+      url: /packages/macros/
+    - title: Server
+      url: /packages/server/
+    - title: Networking
+      url: /packages/networking/
+    - title: Testing
+      url: /packages/testing/
+    - title: Debug Logging
+      url: /packages/logging/
```

The package pages are broken currently, and need the `.html` suffix instead:

```suggestion
  pages:
    - title: Community Showcase
      url: /packages/showcase.html
    - title: Packages with Macros
      url: /packages/macros.html
    - title: Server
      url: /packages/server.html
    - title: Networking
      url: /packages/networking.html
    - title: Testing
      url: /packages/testing.html
    - title: Debug Logging
      url: /packages/logging.html
```